### ISSUE TITLE
[New game]: Dragon-s-World-Game

### ISSUE THREAD

**Issue by rt-001** [kunjgit/GameZone#41]
### 🎮 Game Request

Game logic and basic description
It is a game in which one dragon tries to save himself from the other dragon.

The provided code allows players to control a dragon character using the arrow keys. The up arrow triggers a jumping animation, while the left and right arrows enable horizontal movement. The code continuously checks the positions of the dragon and an obstacle. If a collision is detected, a game over message is shown. However, if the dragon successfully crosses the obstacle without colliding, the player's score increases by 1. The code also increases the obstacle's speed over time, making the game progressively more challenging. Overall, the code creates a basic gameplay experience where players control a dragon, avoid obstacles, and try to achieve a high score.
### Point down the features

Game points
Single Player Game.
Simple rules.
TechStack
1) Javascript
2) HTML
3) CSS
### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

**Comment by kunjgit** [comment]
Hey @rt-001 can you elaborate about the features !!

**Comment by rt-001** [comment]
@kunjgit  Done

**Comment by Lavesh12** [comment]
Hello,
I hope you're doing well. I am interested in contributing to this project as part of GSSoC. It aligns with my skills and interests, and I believe I can make a meaningful contribution to the project Could you please assign this task to me? 
Looking forward to hearing from you.
Thank you,
Lavesh Vishwakarma

**Comment by rt-001** [comment]
@kunjgit assign this to me.

**Comment by kunjgit** [comment]
Hello @rt-001, Time's Uppp!⏰ 
 Sorry for closing your issue! 
 But it's more than a week since we haven't received anything from your side 😢 . 
 Come up with new ideas, create a new issue and make sure you finish it within a week! 🔥 
 All the best! 🚀 
 Happy Hacking! 💗

**Comment by kunjgit** [comment]
Hey @rt-001 ! Thank you so much for your raising the issue💗 
 It’s all yours, you can come anytime again and make some contributions! 🚀 
 Alone, we can do little, but together we can do so much! 😇

**Comment by rt-001** [comment]
@kunjgit Since I am new to GitHub, it was difficult for me to create a new branch for different issues. Usually, I would wait for my current pull request (PR) to get accepted before starting work on another one.
